src/translations/en.json

```
@@ -5207,6 +5207,14 @@
                   "select-options": {
                     "label": "Select options"
                   },
+                  "number": {
+                    "label": "Number Input",
```

```suggestion
                    "label": "Number",
```
It's not always an input number.